src/journal-remote/journal-gatewayd.c

```diff
@@ -641,6 +641,66 @@ static int request_handler_fields(
         return MHD_queue_response(connection, MHD_HTTP_OK, response);
 }
 
+static int output_field_name(FILE *f, OutputMode m, const char *d) {
+        if (m == OUTPUT_JSON)
+                return fprintf(f, "{ \"field\" : \"%s\" }\n", d);
```

Comment:
Please do not generate manual JSON in new code. Please use the APIs from json.c

Reply:
(field names might container `"`, which would need escaping here, which the json.c apis will handle for you)